test/testrunner.py

```diff
@@ -561,7 +568,7 @@ def main():
             sm_utils.generate_empty_report(report, test_type, "eia")
     else:
         raise NotImplementedError(
-            f"{test_type} test is not supported. Only support ec2, ecs, eks, sagemaker and sanity currently"
+            f"{test_type} test is not supported. Only support ec2, ecs, eks, sagemaker, telemetry and sanity currently"
```

Comment:
nit: santiy is not actually a supported test type now, so would change this to reflect the new sanity tests you're supporting